cqrs-user-command-service/src/test/java/dev/amir/usercommand/application/service/UserInputPortTest.java

```diff
@@ -19,14 +19,14 @@
 @ExtendWith(MockitoExtension.class)
 class UserInputPortTest {
     @Mock
-    private UserOutputPort userOutputPort;
+    private UserOutputPort userOutputPortMock;
     @Mock
-    private UserMessageOutputPort userMessageOutputPort;
+    private UserMessageOutputPort userMessageOutputPortMock;
     @InjectMocks
-    private UserUseCases userUseCases;
+    private UserUseCases userUseCasesMock;
```

Comment:
this is not a mock object, this is the object under test, so, do not include the suffix `Mock`

```suggestion
    private UserUseCases userUseCases;
```